package.json

```diff
@@ -39,6 +39,12 @@
     "build:assets:other": "cpy --flat 'src/assets' '!**.svg' '!**.tsx' assets",
     "build:assets:svg": "svgo --folder src/assets -o assets",
     "build:react": "tsup src/react/index.ts --dts --format=esm,cjs --sourcemap --out-dir=react",
+    "build:src": "yarn run build:src:logo && yarn run build:src:favicon:png && yarn run build:src:favicon:ico && yarn run build:src:android-icon && yarn run build:src:apple-touch-icon",
+    "build:src:android-icon": "convert src/assets/favicon.png \\( +clone -resize 192x192 -write src/assets/icon-192.png +delete \\) -resize 512x512 src/assets/icon-512.png",
```

Comment:
Normally I'd quibble about referencing global scripts like `convert` and `rsvg` that are assumed to be on the host machine but probably aren't. However, I'm glad you're showing your work. We can later swap these scripts out with a CLI packaged as an NPM dependency.

Reply:
Yep. I had to remove those runs from the CI build chain because the GitHub runners don't have imagemagick convert or rsvg-convert available. But this at least establishes a more repeatable process.

The logo-icon.svg and favicon.svg are generated, but the other svgs are not yet generated. Their paths were manually copied from `logo-icon.svg`.